site/src/styles/global.css

```
@@ -870,3 +870,200 @@ body {
+   back: unlayered CSS beats every layer.
+
+   The radius travels as a custom property because the canvas is created by
+   the library at a depth we do not control, and `inherit` only reads one
+   level up. clip-path is the belt to that pair of braces: it clips
+   descendants whatever their overflow, z-index or containment.
+---------------------------------------------------------------- */
+/* The shadow lives on the shell, outside the clip, or the frame's own
+   clip-path would cut it off. */
+.feat-frame-shell {
+  --frame-radius: 28px;
+  border-radius: var(--frame-radius);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)